crates/api_common/src/utils.rs

```diff
@@ -472,6 +472,20 @@ pub fn send_application_approved_email(
   send_email(&subject, email, &user.person.name, &body, settings)
 }
 
+
+/// IMPLEMENT ME
+pub fn send_application_denied_email(
+  user: &LocalUserView,
+  settings: &Settings,
+) -> Result<(), LemmyError> {
+  let email = &user.local_user.email.clone().expect("email");
```

Comment:
Dont use expect.

Reply:
You'll see this was copy-pasted from the approval send, which also unfortunately uses expect. 

It does check for the email before this is called, but I agree, using if let is safer.